packages/core/parcel/src/cli.js

```diff
@@ -75,6 +75,11 @@ const commonOptions = {
   '--cache-dir <path>': 'set the cache directory. defaults to ".parcel-cache"',
   '--watch-dir <path>':
     'set the root watch directory. defaults to nearest lockfile or source control dir.',
+  '--watcher-ignore-dirs [path]': `list of directories watcher should not be tracking for changes. defaults to ['.git', '.hg']`,
+  '--watcher-backend': new commander.Option(
+    '--watcher-backend <name>',
+    'set watcher backend',
+  ).choices(['fs-events', 'watchman', 'inotify', 'windows', 'brute-force']),
```

Comment:
wonder if the allowed options should depend on the platform parcel is running on. e.g. inotify is only available on linux, fs-events is only available on Mac, etc.